foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_9_0/MLFlowDockerfileMigration.java

```diff
@@ -45,11 +46,10 @@ protected boolean shouldExecuteOnFile(File file) {
         boolean shouldExecute = false;
         try (BufferedReader dockerfileReferenceConfig = new BufferedReader((new FileReader(file)))) {
             String line;
-            while((line = dockerfileReferenceConfig.readLine()) !=null && !shouldExecute) {
-                for (String key : DOCKER_IMAGE_REFERENCE_MAP.keySet()) {
-                    if (line.contains(key)) {
-                        shouldExecute = true;
-                    }
+            while ((line = dockerfileReferenceConfig.readLine()) != null) {
```

Comment:
I (Informational): updated to only look for the `boozallen/aissemble-mlflow:1.7.0` that we want to migrate out